.rhdh/scripts/prepare-restricted-environment.sh

```diff
@@ -800,15 +852,18 @@ function mirror_image_to_registry() {
   src_image=$(replaceInternalRegIfNeeded "$1")
   local dest_image
   dest_image=$2
+  
   echo "Mirroring $src_image to $dest_image..."
   skopeo copy --preserve-digests --remove-signatures --all --dest-tls-verify=false docker://"$src_image" docker://"$dest_image"
 }
 
 function mirror_image_to_archive() {
   local src_image
+  local dest_image
   dest_image=$(replaceInternalRegIfNeeded "$1")
```

Comment:
```suggestion
  src_image=$(replaceInternalRegIfNeeded "$1")
```

I meant not using `dest_image` at all, since it is not needed.